responses/edge.go

```diff
@@ -0,0 +1,9 @@
+package responses
+
+import "github.com/uber/gonduit/entities"
+
+// EdgeSearchResponse represents a response of the edge.search call.
+type EdgeSearchResponse struct {
+	Data   []entities.Edge `json:"data"`
```

Comment:
Looks like EdgeType is actually used for responses - library might fail if phabricator is upgraded and new type of edge is returned... Not very graceful way to handle. Please check what would happen if unknown edgetype is returned by phabricator - if parsing doesn't crash - then it is fine...

Reply:
It won't fail as string of `EdgeType` will be returned.